Rakefile

```diff
@@ -4,3 +4,18 @@
 require File.expand_path('../config/application', __FILE__)
 
 Rails.application.load_tasks
+
+# Create a new DB backup, and import it to the local Postgres DB
+task :db do
+  app = ENV["HEROKU_APP"] || "fastlane-refresher"
+  db_name = ENV["DB_NAME"] || "refresher"
+
+  puts "This script is going to drop your local database #{db_name} and fetch the database from heroku #{app}. Please confirm!"
+  STDIN.gets
```

Comment:
I think someone might reasonably expect that if they entered 'n' here, that things would stop.

Should the warning be changed to say "kill the process now" or similar? Or should 'y', 'n' be the only valid inputs?